app/src/main/java/com/microsoft/office365/snippetapp/Snippets/EmailSnippets.java

```diff
@@ -250,6 +255,34 @@ public Boolean addItemAttachment(
     }
 
     /**
+     * Deletes all attachments from a message attachment collection and update
+     * the message in the Drafts folder
+     *
+     * @param mailId  The id of the message to update
+     * @return boolean. Success flag. True if attachments are deleted
```

Comment:
Recommend changing to void return type since this no longer checks if attachments are deleted.